app/uk/gov/hmrc/cdsreimbursementclaimfrontend/models/DocumentTypeRejectedGoods.scala

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cdsreimbursementclaimfrontend.models
+
+import cats.Eq
+import julienrf.json.derived
+import play.api.libs.json._
+
+sealed trait DocumentTypeRejectedGoods
+
+object DocumentTypeRejectedGoods {
+
+  case object Foo extends DocumentTypeRejectedGoods
```

Comment:
Can we find out the document types before we merge this?

Reply:
not sure if we need, this will not affect the journey